helm/cloudnative-pg/templates/cilium-network-policy.yaml

```diff
@@ -15,6 +15,15 @@ spec:
     # Allow egress to Kubernetes API server.
     - toEntities:
         - kube-apiserver
+        - cluster
```

Comment:
Just for my knowledge, why is `cluster` needed in addition to the specific matched endpoints?

Reply:
It isn't. Sorry my bad, I had forgot to remove this from some previous tests.